tests/engine-state/detect-engine-state-03/suricata.yaml

```diff
@@ -0,0 +1,14 @@
+%YAML 1.1
+---
+
+outputs:
+  - eve-log:
+      enabled: yes
+      types:
+        - files
+        - stats
```

Comment:
It would be useful to log at least `http`, `anomaly` and `flow` events alongside these. If the file events are not coming through, we'll get some idea.

Reply:
You mean to add http, anomaly and flow to suricata.yaml or to createst?